beacon_node/operation_pool/src/lib.rs

```diff
@@ -1529,7 +1529,8 @@ mod release_tests {
         let (harness, _) = sync_contribution_test_state::<MainnetEthSpec>(1).await;
 
         let op_pool = OperationPool::<MainnetEthSpec>::new();
-        let state = harness.get_current_state();
+        let mut state = harness.get_current_state();
+        state.update_pubkey_cache().unwrap();
```

Comment:
@michaelsproul could not figure out where the caches are lost. The genesis state in the harness builds the caches but they are lost along the way. Maybe while advancing the blocks?

Reply:
Not sure, can dig into this if it's still a problem later